code/modules/power/supermatter/supermatter.dm

```diff
@@ -1,120 +1,3 @@
-//Ported from /vg/station13, which was in turn forked from baystation12;
-//Please do not bother them with bugs from this port, however, as it has been modified quite a bit.
-//Modifications include removing the world-ending full supermatter variation, and leaving only the shard.
-
-//Zap constants, speeds up targeting
-
-#define BIKE (COIL + 1)
-#define COIL (ROD + 1)
-#define ROD (LIVING + 1)
-#define LIVING (MACHINERY + 1)
-#define MACHINERY (OBJECT + 1)
-#define OBJECT (LOWEST + 1)
-#define LOWEST (1)
-
```

Comment:
God damn it you're right, don't move this comment and these defines tho, they should both be local to this file.

Reply:
Alright, moved it around. Hope this is fine now